src/app/src/main/kotlin/org/imperial/mrc/hint/controllers/PasswordController.kt

```diff
@@ -36,9 +38,9 @@ class PasswordController(private val userRepository: UserRepository,
 
             emailManager.sendEmail(emailMessage, email)
 
-            return token
+            return SuccessResponse(token).toJsonString()
         }
 
-        return ""
+        return SuccessResponse(true).toJsonString()
```

Comment:
I know you're working on this code now so this is liable to change, just updated these for now so they conform to the response schema.